demos/toolbar/default-toolbar.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<!--
+  Copyright 2016 Google Inc. All rights reserved.
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+      https://www.apache.org/licenses/LICENSE-2.0
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License
+-->
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>MDC Toolbar Demo</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
+    <style>
+      body {
+        margin: 0px;
+        font-size: 16px;
+      }
+      p {
+        margin: 0px;
+        padding: 20px 16px;
+      }
+    </style>
+    <script src="../assets/material-components-web.css.js" charset="utf-8"></script>
+  </head>
+  <body>
+    <main>
+      <div class="mdc-toolbar mdc-toolbar_fixed mdc-theme--primary-bg mdc-theme--text-primary-on-primary">
```

Comment:
Rather than using the `mdc-theme--*` classes directly on toolbar, I'd prefer if toolbars were made theme-aware at the component-level, as is the convention with all of our other components. The reason for this is that users then have to add the minimal amount of CSS classes onto the component for it to be styled properly. E.g. when using `mdc-button` with a `--primary` or `--secondary` modifier class, it knows intrinsically to make its BG color the theme's primary (or secondary) color, as well as to use the proper text color for that theme. 

The convention we use in mdc-web is to use the [mdc-theme mixins](https://github.com/material-components/material-components-web/tree/master/packages/mdc-theme#mdc-theme-prop-mixin) within the component sass to achieve this. You can find examples in almost all of our component packages.